### DISCUSSION TITLE
Friesach and Tremulants

### DISCUSSION THREAD

**Discussion by jason-barba** [GrandOrgue/grandorgue#1732] [Help]
It seams that the Tremolo II in Friesach do not work in my setup. Please, advice me were to start looking. Some setting? Or have I not loaded all sample files?
I have not loaded the Solowerk sample files due to lack of memory. 

//Jens

**Comment by ClarkeBrunt** [comment]
I have all of Friesach loaded (16GB memory, lossless compression enabled). I'm struggling to convince myself that Tremolo II (i.e. for 2nd manual, Schwellwerk) is operating. I think it is, but it's barely perceptible. Whereas Tremolo III (for 3rd manual, Solowerk) is clearly audible.

I just had a brief look in the .organ file. Couldn't see anything obviously wrong. Settings for both tremulants look similar. When I've a bit more time, I might try editing the .organ file to make Tremolo II really obvious (i.e. large ampliture) so can definitely tell wheher it is working.

**Reply by jason-barba** [reply]
Good to know I am not alone with this problem ;-)

**Comment by eturpault** [comment]
In the file Friesach.organ :
Tremolo II  (Schwellwerk) has : Period=250, StartRate=50, StopRate=30, AmpModDepth=6
Tremolo III (Solowerk) __ has : Period=208, StartRate=30, StopRate=70, AmpModDepth=8
I find weird that in Tremolo II the stop rate (stop time) is lower than the start rate (startup time). 
Try giving to Tremolo II the attributes of Tremolo III (or other values) and test if it is better for you.

**Reply by jason-barba** [reply]
As I understand the help, the start and stops rate/time are individual settings. I might be wrong ;-)
My first try would be to increase the AmpModDepth as suggested by @ClarkeBrunt.

**Reply by ClarkeBrunt** [reply]
I just tried changing AmpModDepth to 30 for both tremulants. Both were definitely working, and (as expected) fairly objectionable! So it just seems that AmpModDepth=6 isn't enough - unless the actual Friesach organ is like this. So something greater than 6 (but less than 30!) should be good.

The help on the settings for a synthesized tremulant in the .organ file says:

Period (integer 32 - 44100, required) Period of the tremulant in ms 
StartRate (integer 1 - 100, required) Determines the startup time of the tremulant. 
StopRate (integer 1 - 100, required) Determines the stop time of the tremulant. 
AmpModDepth (integer 1 - 100, required) Determines, how much the volume will be changed. 

So Period is clear. AmpModDepth is probably a percentage. And it isn't clear what the units of StartRate/StopRate are, or exactly what they do, but I imagine they are how quickly the tremulant 'gets going' when turned on, and how quickly it stops when turned off (assuming we're emulating some kind of mechanical device which alters the wind supply).

I recall that the swell pedal doesn't have much effect either, so maybe another candidate for increasing the amplitude.

**Reply by jason-barba** [reply]
Thanks. Will test. If I remember correctly, all I have to do is export the settings, close GO, edit the .organ fil, restart GO, load Friesach and answer yes on an error message, and finale import the settings again. Am I right?

**Reply by ClarkeBrunt** [reply]
You probably remember better than me how best to retain your settings. It's as well to do an export, so you've got the settings in a saved file with a name of your choosing (essentially a backup), but I think this only saves what's already in the CMB file - the file for which you probably have to confirm that you don't mind that the organ file has changed. So probably no need to import your saved settings.

**Reply by jason-barba** [reply]
Finally! I have increased the AmpModDepth from 6 to 10 for [Tremulant001]. I am satisfied.
Sorry for the delay, but I had to be sure the organ was working so no testing. Up until now ;-)
